### ISSUE TITLE
Hidden payload field not submitted to the server

### ISSUE THREAD

**Issue by bramley** [altcha-org/altcha#4]
I am having difficulty getting the hidden field containing the payload to be sent to the server when submitting. In the browser console I can see that the verification was successful 

```
ALTCHA using provided json data altcha.js:757:97
ALTCHA challenge 
Object { algorithm: "SHA-256", challenge: "65b21b970585fe54b0e82a0c7ebaaea0aca92e7aa45d924d18c5719fbb13730b", salt: "1870b01d8715042f7a9df1168ee969c4", signature: "bbedba7edf53774828537987be5486103fabf35c9a3ffd775d6da98467559ec6" }
altcha.js:757:97
ALTCHA solution 
Object { number: 293, took: 24, worker: true }
altcha.js:757:97
ALTCHA verified altcha.js:757:97
ALTCHA payload eyJhbGdvcml0aG0iOiJTSEEtMjU2IiwiY2hhbGxlbmdlIjoiNjViMjFiOTcwNTg1ZmU1NGIwZTgyYTBjN2ViYWFlYTBhY2E5MmU3YWE0NWQ5MjRkMThjNTcxOWZiYjEzNzMwYiIsIm51bWJlciI6MjkzLCJzYWx0IjoiMTg3MGIwMWQ4NzE1MDQyZjdhOWRmMTE2OGVlOTY5YzQiLCJzaWduYXR1cmUiOiJiYmVkYmE3ZWRmNTM3NzQ4Mjg1Mzc5ODdiZTU0ODYxMDNmYWJmMzVjOWEzZmZkNzc1ZDZkYTk4NDY3NTU5ZWM2IiwidG9vayI6MjR9

```
and also see the hidden field in the inspector

`<input type="hidden" name="altcha2" value="eyJhbGdvcml0aG0iOiJTSEEtMjU2IiwiY2hhbGxlbmdlIjoiNjViMjFiOTcwNTg1ZmU1NGIwZTgyYTBjN2ViYWFlYTBhY2E5MmU3YWE0NWQ5MjRkMThjNTcxOWZiYjEzNzMwYiIsIm51bWJlciI6MjkzLCJzYWx0IjoiMTg3MGIwMWQ4NzE1MDQyZjdhOWRmMTE2OGVlOTY5YzQiLCJzaWduYXR1cmUiOiJiYmVkYmE3ZWRmNTM3NzQ4Mjg1Mzc5ODdiZTU0ODYxMDNmYWJmMzVjOWEzZmZkNzc1ZDZkYTk4NDY3NTU5ZWM2IiwidG9vayI6MjR9">`

but the http request contains only my own form fields, some of which are also hidden.

Is there something that I am misunderstanding?

**Comment by ovx** [comment]
Hi, sorry for that, it was a bug in the previous version; Its fixed in 0.1.2.

**Comment by bramley** [comment]
Thanks! Working now.
